### DISCUSSION TITLE
posts/Go/golang-unit-test-5/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#319] [Announcements]
https://www.liwenzhou.com/posts/Go/golang-unit-test-5/ 

 李文周的博客介绍了Go语言测试,单元测试怎么写,go test 执行测试,goconvey教程,测试结果的webui页面自动测试，支持图形化测试,如何编写单元测试

**Comment by daniel-style** [comment]
请问这是什么留言系统

**Comment by ForeverSRC** [comment]
第6篇没有评论我就在这写。第6篇提到的三个问题正好是最近开发中遇到的，真是太有感触了!
